<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopMenu</class>
 <widget class="QWidget" name="ShopMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="continue_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/continue.png</normaloff>:/res/continue.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="increase_ore_value_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>矿更值钱</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/increase_item_value.png</normaloff>:/res/increase_item_value.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>187</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>永久升级：500 $</string>
   </property>
  </widget>
  <widget class="QToolButton" name="increase_ore_num_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>更多矿</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/increase_mine.png</normaloff>:/res/increase_mine.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="belt_upgrade_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>升级传送带</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/belt_button.png</normaloff>:/res/belt_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="miner_upgrade_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>升级采矿机</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/miner_button.png</normaloff>:/res/miner_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="cutter_upgrade_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>升级切割机</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/cutter_button.png</normaloff>:/res/cutter_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="hub_upgrade_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>加大交付中心</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/hub_big.png</normaloff>:/res/hub_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>468</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>临时升级（下一关时会消失）：100 $</string>
   </property>
  </widget>
  <widget class="QLabel" name="coin_count_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>150</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
